frontend/public/components/utils/breadcrumbs.ts

```diff
@@ -12,10 +13,10 @@ export const getBreadcrumbPath = (match: any, customPlural?: string) => {
   return `/k8s/cluster/${customPlural || match.params.plural}`;
 };
 
-export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any) => () => [
+export const breadcrumbsForDetailsPage = (kindObj: K8sKind, match: any, t: TFunction) => () => [
```

Comment:
It's better to not pass `t` around since `useTranslation` can be initialized with different values. I'd just use the global i18n instance in this case.